content/ytt/docs/latest/how-it-works.md

```diff
@@ -0,0 +1,150 @@
+---
+title: "How it works"
+---
+
+## Overview
+
+Let's get an idea of how `ytt` works by looking at the high-level concepts and flow.
+
+## The Pipeline
+
+When you invoke `ytt` ...
+
+```console
+$ ytt -f (input files)
```

Comment:
should this command look a bit more real. it looks a bit confusing since () mean something else in bash.